libs/libxslt/Makefile

```diff
@@ -22,6 +22,7 @@ PKG_LICENSE:=MIT
 PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:xmlsoft:libxslt
 
+PKG_FIXUO:=autoreconf
```

Comment:
PKG_FIXUP ?

Reply:
Yikes, yeah. I wrote it properly in my VM.